front-end/src/components/EditablePostContent.tsx

```diff
@@ -91,17 +98,117 @@ const EditablePostContent = ({ className, isEditing, isTipProposal, onchainId, p
 		}
 	});
 
+	const [discussionPostQuery, { data, loading }] = useDiscussionPostAndCommentsLazyQuery();
+
+	const handleLink = () => {
+		if (!linkPostId) {
+			setValidationError('Please enter a valid post id');
+		}
+
+		if (isNaN(Number(linkPostId))) {
```

Comment:
we can replace this with `if(Number.isNaN(linkPostId)` it checks for both number and NaN